docs/backend_readme.md

```diff
@@ -235,4 +235,28 @@ To track the historical events for a certain model based on their id and timesta
 Context.objects.filter(metadata__tracked_model_id=tracked_model_id).order_by("metadata__timestamp")
 ```
 
-</details>
+---
+
+### Troubleshooting
+
+#### Aliases
+
+For a list of command aliases, refer to [.bash_aliases](https://github.com/BetterAngelsLA/monorepo/blob/main/.bash_aliases)
+
+#### Resetting DB
```

Comment:
Good call on the title!

And yeah, the only reason why I added the migrate step was because it was required for the backend to run after the reset. Would you still feel more inclined to make it a usage note instead?

Reply:
I think it's typically better to document the item individually first. So we document that alias X does Y (and only Y).
And then similar to [this doc](https://django-extensions.readthedocs.io/en/latest/reset_db.html) we can provide additional info that may be useful.

Reply:
Updated it again. Keep letting me know if the format or wording isn't up to standard, and I'll keep trying to adjust it to get it right!